### DISCUSSION TITLE
Deprecating `babel-eslint`

### DISCUSSION THREAD

**Discussion by brettz9** [babel/babel#12702] [Q&A] [answered]
Hi,

Is there a reason `babel-eslint` is not deprecated (and have a message on the old npm README as opposed to just the https://github.com/babel/babel-eslint README) if `@babel/eslint-parser` is?

**Comment by kaicataldo** [comment, marked as answer]
Thanks for bringing this to our attention! We probably should `npm-deprecate` the package so that folks have a clearer upgrade path. Will discuss with the rest of the team and circle back.

**Comment by kaicataldo** [comment]
`babel-eslint` is now deprecated on `npm`. Thanks again for bringing this to our attention!
